files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -156,6 +156,10 @@
   cert, key = OmnibusHelper.gen_certificate
 end
 
+# TODO: we are now using public_keys instead of certs and the
+#       bootstrap process now uses the public key. This is still
+#       here for backwards compatibility in case it is used anywhere
+#       else but we should consider deprecating it in the future.
```

Comment:
@sdelano Where does erchef use this?  I didn't find anything via a simple grep.  I'm confused why we would ever load the public key from disk rather than from the database.